packages/providers/audit-logs-local/lib/index.js

```diff
@@ -2,6 +2,9 @@
 
 const auditLogContentType = require('./content-types/audit-log');
 
+// @TODO: Hardcoded for now, we should get this from the config later
```

Comment:
I don't think the user would be able to configure this 🤔 but maybe it's something Yannis can clarify